plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -1157,24 +1169,24 @@ func (c *PodAffinityChecker) satisfiesExistingPodsAntiAffinity(pod *v1.Pod, meta
 		filteredPods, err := c.podLister.FilteredList(nodeInfo.Filter, labels.Everything())
 		if err != nil {
 			glog.Errorf("Failed to get all pods, %+v", err)
-			return false
+			return false, nil
 		}
 		if matchingTerms, err = c.getMatchingAntiAffinityTerms(pod, filteredPods); err != nil {
 			glog.Errorf("Failed to get all terms that pod %+v matches, err: %+v", podName(pod), err)
-			return false
+			return false, ExistingPodAffinityNotMatchErr
```

Comment:
Return false, nil here and other cases below.

Reply:
why not return ErrExistingPodsAntiAffinityRulesNotMatchand specific error instead of nil?